doc: i.MX updated doc for Zephyr release 1.13

diegosueiro opened the pull request:
i.MX updated doc section for Zephyr release 1.13:

Architectures:
    nxp_imx: Added RDC (Resource Domain Controller) configuration for i.MX SoCs
    to share peripherals between the Cortex A and Cortex M4 cores.


Signed-off-by: Diego Sueiro <diego.sueiro@gmail.com>

dbkinder commented on doc/release-notes-1.13.rst:
```diff
@@ -21,14 +21,18 @@ Kernel
 Architectures
 *************
 
+* nxp_imx: Added RDC (Resource Domain Controller) configuration for i.MX SoCs to share peripherals between the Cortex A and Cortex M4 cores.
```
Please keep line length < 80 chars please.  Also your commit message has a line that's too long (see https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/21594/5/tests)

diegosueiro committed:
doc: i.MX updated doc sections for Zephyr release 1.13:

Architectures:
nxp_imx: Added RDC (Resource Domain Controller) configuration for i.MX
SoCs to share peripherals between the Cortex A and Cortex M4 cores.

Signed-off-by: Diego Sueiro <diego.sueiro@gmail.com>

diegosueiro commented:
recheck
